src/angular/components/send/send.component.ts

```diff
@@ -199,7 +215,7 @@ export class SendComponent implements OnInit {
 
     private applyTextSearch() {
         if (this.searchText != null) {
-           this.filteredSends = this.searchService.searchSends(this.filteredSends, this.searchText);
+            this.filteredSends = this.searchService.searchSends(this.filteredSends, this.searchText);
```

Comment:
this looks accidental

Reply:
It's a linter fix. One of the old tabs had only 3 spaces